subsys/bluetooth/host/classic/rfcomm.c

```diff
@@ -37,7 +37,7 @@ LOG_MODULE_REGISTER(bt_rfcomm);
 #define RFCOMM_MIN_MTU		BT_RFCOMM_SIG_MIN_MTU
 #define RFCOMM_DEFAULT_MTU	127
 
-#define RFCOMM_MAX_CREDITS		(CONFIG_BT_BUF_ACL_RX_COUNT - 1)
+#define RFCOMM_MAX_CREDITS		(BT_BUF_ACL_RX_COUNT - 1)
```

Comment:
Would it make sense to set this as 
```suggestion
#define RFCOMM_MAX_CREDITS		(CONFIG_BT_MAX_CONN)
```

Reply:
Right, for some reason I imagined that `BT_BUF_ACL_RX_COUNT` was always `CONFIG_BT_MAX_CONN + 1` but that's just the default